spec/models/bet_spec.rb

```diff
@@ -12,28 +12,31 @@
   end
 
   let(:bet1) do
-    Bet.create(bet: 'Superbowl: Atlanta vs New England, place your bets!',
+    Bet.create(bet_description: 'Superbowl: Atlanta vs New England,
+               place your bets!',
                expires_at: DateTime.tomorrow, creator_id: bob.id)
   end
-  let(:option1) { BetOption.create(option_text: 'Atlanta wins', bet: bet1) }
+  let(:option1) do
+    BetOption.create(option_text: 'Atlanta wins', bet: bet1)
+  end
```

Comment:
I try to keep my 'old code cleanup' tasks out of my feature PRs, and submit them as separate PRs :)